packages/xo-server/src/xapi/index.js

```diff
@@ -1437,7 +1437,25 @@ export default class Xapi extends XapiBase {
     return /* await */ this._snapshotVm(this.getObject(vmId), nameLabel)
   }
 
-  async _startVm(vm, host, force) {
+  async _startVm(vm, host, force = false) {
+    if (!force) {
+      const runningVmsVifsByMac = groupBy(
+        flatMap(
+          filter(this.objects.all, {
+            $type: 'VM',
```

Comment:
We should omit the passed `VM`.

Reply:
it's already a non-running VM but maybe you' re right because we don't check the VM `power-state`.

Reply:
Ah, my bad you're right.